adapters/ix/ix.go

```diff
@@ -309,3 +326,34 @@ func marshalJsonWithoutUnicode(v interface{}) (string, error) {
 	// https://pkg.go.dev/encoding/json#Encoder.Encode
 	return strings.TrimSuffix(sb.String(), "\n"), nil
 }
+
+func BuildIxDiag(request *openrtb2.BidRequest) error {
+	extRequest := &ExtRequest{}
+	if request.Ext != nil {
+		if err := json.Unmarshal(request.Ext, &extRequest); err != nil {
+			return err
+		}
+	}
+	ixdiag := &IxDiag{}
+
+	if extRequest.Prebid != nil {
+		ixdiag = &IxDiag{
+			PbjsV: extRequest.Prebid.Channel.Version,
```

Comment:
If condition in line 339 is true, I believe we are overwritting `ixdiag`'s original value. Can we assign `PbjsV`'s value as we do in lines 347 and 349 for `PbsV` instead?
```
337     ixdiag := &IxDiag{}
338
339     if extRequest.Prebid != nil {
340 -       ixdiag = &IxDiag{
341 -           PbjsV: extRequest.Prebid.Channel.Version,
342 -       }
    +       ixdiag.PbjsV = extRequest.Prebid.Channel.Version
343     }
344
345     // Slice commit hash out of version
346     if strings.Contains(version.Ver, "-") {
347         ixdiag.PbsV = version.Ver[:strings.LastIndex(version.Ver, "-")]
348     } else if version.Ver != "" {
349         ixdiag.PbsV = version.Ver
350     }
adapters/ix/ix.go
```

Reply:
thanks, updated